doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/launching-clusters/on-premises.md

```diff
@@ -0,0 +1,116 @@
+# Launching On-Premises Cluster
```

Comment:
Can we delete the old file? https://ray--27431.org.readthedocs.build/en/27431/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/running-ray-cluster-on-prem.html